pytensor/graph/basic.py

```diff
@@ -558,6 +558,46 @@ def get_parents(self):
             return [self.owner]
         return []
 
+    def convert_string_keys_to_pytensor_variables(self, inputs_to_values):
```

Comment:
Given we are in `pytensor`, no need to mention it in the name. If you want you can specify the return type is `Dict[Variable, Variable]`

```suggestion
    def convert_string_keys_to_variables(self, inputs_to_values):
```